docs/ref/models/querysets.txt

```diff
@@ -2171,7 +2171,7 @@ This has a number of caveats though:
 * It does not work with child models in a multi-table inheritance scenario.
 * If the model's primary key is an :class:`~django.db.models.AutoField`, the
   primary key attribute can only be retrieved on certain databases (currently
-  PostgreSQL and MariaDB 10.5+). On other databases, it will not be set.
+  PostgreSQL, MariaDB 10.5+ and SQLite 3.35+). On other databases, it will not be set.
```

Comment:
Comma is missing before 'or'/'and' (see https://en.wikipedia.org/wiki/Serial_comma).

```suggestion
  PostgreSQL, MariaDB 10.5+, and SQLite 3.35+). On other databases, it will not be set.
```